draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2883,7 +2889,125 @@ https://video.example.com/k/movie1/4/013.ts
+dCDN MAY reject such requests for urgent processing using “ereject” error code. This can happen due to dCDN internal business logic (e.g. batch-driven purge and invalidation), or due to dependency on other triggers that could not be completed by dCDN immediately. In such a case uCDN should either modify the request removing prerequisites or cancel such pending triggers before re-trying the request.
+		</t>
+
+
+		</list>
+
+	</t>
+	<t>
+	uCDN may combine multiple options in the same trigger command. dCDN should consider the following when processing such commands:
+	<list style="symbols">
+	<t>(i) dCDN MUST reject triggers that are dependent on other pending triggers with lower priority to prevent deadlocks.</t>
+	<t>(ii) When introducing an urgent trigger uCDN SHOULD indicate priority that is equal to or higher than the highest priority among the pending triggers that belong to the same uCDN. dCDN MUST reject urgent triggers that have lower priority than other pending triggers of the same uCDN.</t>
+	<t>(iii) dCDN MUST reject an urgent trigger that has pending triggers as its prerequisite. dCDN MAY reject an urgent trigger that has triggers in “active” or “cancelling” state as its prerequisite, for example when such triggers are deemed to take a long time to complete or to cancel.</t>
+	</list>
+	
+	</t>
+	<t>
+	Object specification:
+
+        <t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: priority">
+        <vspace blankLines="1"/>
+	Description: Relative weight of the trigger. 
+	When picking a trigger for execution from all pending triggers posted by each uCDN, 
+	the dCDN MUST choose the trigger with the highest priority first. 
+        <vspace blankLines="1"/>
+	Value: Integer from 0 to 100.
+        <vspace blankLines="1"/>
+	Mandatory: No. The value defaults to zero if omitted.
+        </t>
+        </list></t>
+        </list></t>
+
+
+        <t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: depends">
+        <vspace blankLines="1"/>
+	Description: Links to Trigger Status Resources that the current Trigger Command depends on.
+	Indicates which triggers should fully finish processing before starting execution of the current trigger. 
+	The triggers need to be in one of the following states to be considered finished: 
+	“complete”, “processed”, “failed” or “cancelled”.
+        <vspace blankLines="1"/>
+	Value: A JSON array of zero or more URLs, represented as JSON strings.
+        <vspace blankLines="1"/>
+	Mandatory: No.
```

Comment:
Should we indicate that this defaults to an empty list (or no dependencies) if omitted?

Reply:
Added "In case of a missing or an empty list, no dependencies are assumed."